__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+beforeEach(() => {
+    return seed(testData)
+})
+describe("GET /api/categories", () => {
+  test("returns an array of category objects, with slug and description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(Object.keys(body)).toEqual(["categories"]);
+
+        const { categories } = body;
+        expect(categories).toBeInstanceOf(Array);
+
```

Comment:
This is looking good, the only thing I might have added is a check to ensure that the length of the array is more that 0 just because you are forEach-ing over the response so if the array is empty there's nothing to for each over